config.js

```diff
@@ -8,15 +8,15 @@ const __DEV__ = env === 'development'
 const __STAGING__ = env === 'staging'
 const __TEST__ = env === 'test'
 const __PROD__ = env === 'production'
-const __BASE__ = '/'
+const __BASE__ = __PROD__ ? '/stardust' : ''
```

Comment:
No base href in local dev, which as you'll see in `index.ejs` just omits the base tag altogether.